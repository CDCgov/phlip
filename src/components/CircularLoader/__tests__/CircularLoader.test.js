import React from 'react'
import { shallow } from 'enzyme'
import { CircularLoader } from '../index'

const props = {
  type: 'indeterminate',
  color: 'primary'
}

describe('CircularLoader component', () => {
  test('should render correctly', () => {
    expect(shallow(<CircularLoader {...props} />)).toMatchSnapshot()
  })
})