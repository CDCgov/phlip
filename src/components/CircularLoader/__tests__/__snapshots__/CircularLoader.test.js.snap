// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CircularLoader component should render correctly 1`] = `
<WithStyles
  color="primary"
  type="indeterminate"
/>
`;
